massa-consensus-worker/src/tests/mock_protocol_controller.rs

```diff
@@ -47,10 +51,12 @@ impl MockProtocolController {
     // Note: if you care about the operation set, use another method.
     pub async fn receive_block(&mut self, block: Block) {
         let block_id = block.header.compute_block_id().unwrap();
+        let serialize_block = block.to_bytes_compact().expect("Fail to serialize block.");
```

Comment:
I don't think we care about the serialized forms in most cases, so one could just pass an empty vec to `store_block`.

Reply:
As it's in tests we don't care much about performance and I think it's better to be closed to reality.

Reply:
ok